### ISSUE TITLE
Rename conf/systemd/%run%-ledgersmb to ledgersmb_%run%.service

### ISSUE THREAD

**Issue by hasorli** [ledgersmb/LedgerSMB#2691]
Reason: 
From a branding point of view (and easy to remember for users/sysadmins). 

Supported by: 
dcg_mx 
https://matrix.to/#/!qyoLumPqusaXqFJNyK:matrix.org/$14899431931969151aMHRf:matrix.org
Yves 
https://matrix.to/#/!qyoLumPqusaXqFJNyK:matrix.org/$14899505662029660dDCPG:matrix.org
ehuelsmann 
https://matrix.to/#/!qyoLumPqusaXqFJNyK:matrix.org/$14899475382003640WJAnI:matrix.org

Discussion about support for more than one version in parallel, is a separate issue.
